extra_geom/tests/test_epix100_geometry.py

```diff
@@ -0,0 +1,84 @@
+from extra_geom import Epix100Geometry
+
+import extra_geom
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def epix():
+    return Epix100Geometry.from_origin()
+
+
+def test_epix_geometry(epix):
+    """ Tests that an object was created """
+    assert isinstance(epix, extra_geom.detectors.Epix100Geometry)
+    assert epix.expected_data_shape == (1, 704, 768)
+    assert epix.n_modules == 1
+    assert epix.n_quads == 0
+    assert epix.n_tiles_per_module == 4
+    assert isinstance(epix.modules, list)
+    assert len(epix.modules) == 1
+    assert isinstance(epix.modules[0], list)
+    assert len(epix.modules[0]) == 4
+    for i in range(epix.n_tiles_per_module):
+        assert isinstance(epix.modules[0][0],
+                          extra_geom.detectors.GeometryFragment)
+
+
+def test__tile_silce(epix):
+    _slice = epix._tile_slice(0)
+    assert isinstance(_slice, tuple)
+    assert isinstance(_slice[0], slice)
+    assert isinstance(_slice[1], slice)
+    assert _slice == (slice(352, 704), slice(384, 768))
+
+
+def test_write_crystfel_geom(epix, tmp_path):
+    epix.write_crystfel_geom(tmp_path / 'test.geom')
```

Comment:
As you've included `from_crystfel_geom` in the docs, can we test loading the file back and checking that we get a similar geometry? `test_jungfrau_geometry` has the neatest existing example of this.

Reply:
make sence. done.